ambry-account/src/main/java/com/github/ambry/account/InMemoryUnknownAccountService.java

```diff
@@ -20,18 +20,20 @@
 
 /**
  * An implementation of {@link AccountService} that always has a single entry {@link Account#UNKNOWN_ACCOUNT}. Any
- * queries to this account service will unconditionally return {@link Account#UNKNOWN_ACCOUNT}. This account service
- * is in memory, and does not talk to any persistent storage service.
+ * queries by account name to this account service will unconditionally return {@link Account#UNKNOWN_ACCOUNT}. This
+ * account service is in memory, and does not talk to any persistent storage service.
  */
 class InMemoryUnknownAccountService implements AccountService {
```

Comment:
Have suggested to rename "UNKNOWN_ACCOUNT" to "LEGACY_ACCOUNT" below. On similar lines, would prefer "InMemoryLegacyAccountService".